src/post/post.repository.knex.ts

```diff
@@ -379,6 +392,12 @@ export class KnexPostRepository {
                         (accountId) => accountId,
                     );
                 }
+            } else if (isDeletedPost) {
+                await transaction('posts')
```

Comment:
We discussed this on the call (i think), but we also want to decrement the reply count on `inReplyTo` if it has one